app/src/main/java/org/fossasia/openevent/app/login/LoginActivity.java

```diff
@@ -57,11 +68,30 @@ protected void onCreate(Bundle savedInstanceState) {
         // Notify presenter to attach
         presenter.attach(this);
 
+        checkBoxEnableUrl.setOnCheckedChangeListener((buttonView, isChecked) -> {
+            if(isChecked) {
+                checkBoxEnableUrl.setTextColor(Color.parseColor("#000000"));
```

Comment:
Additionally, please change this all to login presenter interaction. We don't want any logic in our views. This is not MVP